### ISSUE TITLE
Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by rkakash59** [PyTorchLightning/lightning-flash#628]
when I run example code of visualization/pointcloud_detection.py.

I am getting segmentation fault (core dumped) error.

pytorch==1.7.1(pip install wheel from open3d repo)
flash (latest from github repo folder)

**Comment by ethanwharris** [comment]
Hi @rkakash59, thanks for reporting this. Could you share the full error trace that you are getting?

**Comment by rkakash59** [comment]
Hi @ethanwharris 

GPU available: True, used: False
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
/media/drive2/ajk7kor/torch_flash/lib/python3.6/site-packages/pytorch_lightning/trainer/trainer.py:1293: UserWarning: GPU available but not used. Set the gpus flag in your trainer `Trainer(gpus=1)` or script `--gpus=1`.
  "GPU available but not used. Set the gpus flag in your trainer"

  | Name          | Type         | Params
-----------------------------------------------
0 | train_metrics | ModuleDict   | 0
1 | val_metrics   | ModuleDict   | 0
2 | model         | PointPillars | 4.8 M
3 | backbone      | SECOND       | 4.2 M
4 | neck          | SECONDFPN    | 598 K
5 | head          | Conv2d       | 7.7 K
-----------------------------------------------
4.8 M     Trainable params
0         Non-trainable params
4.8 M     Total params
19.342    Total estimated model params size (MB)
/media/drive2/ajk7kor/torch_flash/lib/python3.6/site-packages/flash/core/model.py:399: LightningDeprecationWarning: The `LightningModule.datamodule` property is deprecated in v1.3 and will be removed in v1.5. Access the datamodule through using `self.trainer.datamodule` instead.
  if self.datamodule is not None and getattr(self.datamodule, 'data_pipeline', None) is not None:
/media/drive2/ajk7kor/torch_flash/lib/python3.6/site-packages/flash/core/data/data_pipeline.py:52: UserWarning: Attempted to add a state (CollateFn(collate_fn=<class 'flash.pointcloud.detection.open3d_ml.backbones.ObjectDetectBatchCollator'>)) after the data pipeline has already been initialized. This will only have an effect when a new data pipeline is created.
  " only have an effect when a new data pipeline is created.", UserWarning
/media/drive2/ajk7kor/torch_flash/lib/python3.6/site-packages/pytorch_lightning/trainer/data_loading.py:323: UserWarning: The number of training samples (1) is smaller than the logging interval Trainer(log_every_n_steps=50). Set a lower value for log_every_n_steps if you want to see logs for the training epoch.
  f"The number of training samples ({self.num_training_batches}) is smaller than the logging interval"
Epoch 0: 100%|███████████████████████| 2/2 [00:23<00:00,  7.73s/it, loss=4.96e+03, v_num=12, train_loss_step=4.96e+3, val_loss=2.5e+3]
Segmentation fault (core dumped)

**Comment by ethanwharris** [comment]
Hi @rkakash59, I couldn't replicate the error, so I guess it's an environment issue. Could you try installing flash with:
```
pip install 'git+https://github.com/PyTorchLightning/lightning-flash.git#egg=lightning-flash[pointcloud]'
```

**Comment by ethanwharris** [comment]
Hmm, @rkakash59 can I ask what operating system you are using? It works for me just installing open3d direct from pip

**Comment by rkakash59** [comment]
linux, I can try on windows as well but open3d.ml.torch is not supported in windows.

**Comment by ethanwharris** [comment]
@rkakash59 Linux should be fine. It could be CUDA related as my machine doesn't have GPUs. Could you install the requirements listed here:
https://github.com/isl-org/Open3D-ML/blob/master/requirements-torch-cuda.txt

Otherwise I think you'll have to open an issue of the Open3d github. Sorry I couldn't be of more help

**Comment by Rusteam** [comment]
same issue when trying to run [text classification example](https://lightning-flash.readthedocs.io/en/latest/reference/text_classification.html). 
my env: `cuda==11.7 and nvidia-driver==515.48.07, torch==1.11.0+cu11` on ubuntu 20.04

other `flash` examples do not cause this error, nor does using `transformers` directly
